### DISCUSSION TITLE
Service Naming Convention

### DISCUSSION THREAD

**Discussion by fabianjogschies** [Azure/data-management-zone#121] [General] [answered]
I'm aware of this can be nonsense or an endless discussion.

Some of our customers already got existing naming convention for their services, maybe give them the opportunity to set their own conventions?

**Comment by marvinbuss** [comment, marked as answer]
Thanks for the feedback. If users want to specify their own naming convention, then they are free to do so. If this is something that needs to be done, I would suggest to follow the other deployment option described [here](https://github.com/Azure/data-management-zone/blob/main/docs/EnterpriseScaleAnalytics-CreateRepository.md) and make the necessary changes to the bicep templates as part of the parameter update process.

If we give users the option to change every single name in the deployment, there is a high risk of creating conflicts and there is also the risk of creating a high dropout rate, because users would have to specify names for quite a large number of services. Please let me know, if this would work for your customers. We can also provide guidance on how this can be done.
